web/src/overlays.ts

```diff
@@ -83,6 +83,15 @@ export function settings_open(): boolean {
     return open_overlay_name === "settings";
 }
 
+export function get_user_id_if_user_profile_modal_open(): number {
+    if (is_modal_open() && active_modal() === "#user-profile-modal") {
+        const user_id = $("#user-profile-modal").data("user-id");
+        return user_id ? Number(user_id) : 0;
+    }
+    return 0;
```

Comment:
It'd be preferable to return `null` or `undefined` for this case, rather than 0, which is more in the number space of user IDs.

Reply:
Changed it